rockcraft/oci.py

```diff
@@ -570,11 +582,14 @@ def _add_layer_into_image(
     _process_run([*cmd, "--history.created_by", " ".join(cmd)])
 
 
-def _inject_architecture_variant(image_path: Path, variant: str) -> None:
-    """Inject architecture variant into existing OCI Image config.
+def _inject_manifest(
```

Comment:
```suggestion
def _inject_oci_fields(
```

Probably not the best name, but this function is touching more than just the manifest - it also changes the OCI config.

Reply:
Nice catch.

Reply:
Fixed in 9cc530b.